apps/src/courseExplorer/courseExplorer.js

```diff
@@ -0,0 +1,40 @@
+// Code for the Course/Tools Explorer.  Include it once on the page and it will
+// handle both CourseExplorer and ToolsExplorer on the same page.
+
+var detailRowShowing = -1;
+var toolShowing = -1;
```

Comment:
let instead of var.

Wonder if these should be something like `toolShowingIndex`?